Template10 (Library)/Controls/HamburgerMenu.xaml

```diff
@@ -54,28 +54,45 @@
                 <Setter Property="Width" Value="48" />
             </Style>
 
-            <DataTemplate x:Name="NavButtonTemplate" x:DataType="local:HamburgerButtonInfo">
+            <DataTemplate x:Name="NavButtonContainerTemplate" x:DataType="local:HamburgerButtonInfo">
                 <RadioButton Width="{Binding PaneWidth, ElementName=ThisPage}" MinWidth="48"
                              MaxWidth="{x:Bind MaxWidth, Mode=TwoWay}"
-                             AutomationProperties.Name="Hamburger Menu Item"
                              Command="{Binding NavCommand, ElementName=ThisPage}"
                              CommandParameter="{Binding}" GroupName="Nav" Holding="NavButton_Holding"
                              IsChecked="{Binding IsChecked, Mode=TwoWay}"
                              IsEnabled="{Binding IsEnabled, Mode=TwoWay}" Loaded="NavButton_Loaded"
                              RightTapped="NavButton_RightTapped" Tag="{x:Bind Content}"
                              Tapped="NavButton_Tapped"
-                             ToolTipService.ToolTip="{Binding Path=(ToolTipService.ToolTip)}"
-                             Visibility="{Binding Visibility, Mode=TwoWay}">
+                             ToolTipService.Placement="Top" IsTabStop="True"
+                             Visibility="{Binding Visibility, Mode=TwoWay}"
+                             AutomationProperties.AccessibilityView="Control"
+                             AutomationProperties.Name="{Binding ToolTipText, Mode=OneWay}"
+                             HorizontalAlignment="Stretch">
+                    <RadioButton.Resources>
+                        <converters:DPWhenValueConverter x:Key="DPWhenValueConverter"
+                                            DependencyParameter="{Binding ToolTipText, Mode=OneWay}"
+                                            Otherwise="{x:Null}">
+                            <converters:DPWhenValueConverter.When>
+                                <x:Boolean>False</x:Boolean>
+                            </converters:DPWhenValueConverter.When>
+                        </converters:DPWhenValueConverter>
+                    </RadioButton.Resources>
+                    <ToolTipService.ToolTip>
+                        <Binding Path="IsOpen" ElementName="ThisPage" 
+                                            Converter="{StaticResource DPWhenValueConverter}"/>
```

Comment:
Yuck. Thank goodness that once we are passing {binding} to a ContentPresenter you will not need this horrible approach. No offense. You did solve it. There zero chance I want to introduce a new converter and zero chance I want to introduce using more converters. The performance implication of a converter is the reason. I would rather introduce an internal, new property to the IHamButtonInfo. But, that's moot now, I hope.